compiler/luci/pass/src/PropagateQParamBackwardPass.cpp

```diff
@@ -15,11 +15,201 @@
  */
 
 #include "luci/Pass/PropagateQParamBackwardPass.h"
+#include "QuantizationUtils.h"
 
 #include <luci/IR/CircleNodes.h>
 #include <luci/IR/CircleNodeVisitor.h>
+#include <luci/Service/Nodes/CircleConst.h>
 #include <luci/Log.h>
 
+#include <cmath>
+
+namespace
+{
+
+void quant_const_values(luci::CircleConst *const_node, float scaling_factor, float zerop,
+                        loco::DataType quant_type)
+{
+  uint32_t size = const_node->size<loco::DataType::FLOAT32>();
+
+  const float scaling_factor_inv = 1.0 / scaling_factor;
+  std::vector<int32_t> quantized_values(size);
+  for (uint32_t i = 0; i < size; ++i)
+  {
+    auto data = static_cast<double>(const_node->at<loco::DataType::FLOAT32>(i));
+    double quantized_float = std::round(data * scaling_factor_inv) + zerop;
```

Comment:
Unlike other codes, this variables are declared as `double`, not `float`.
Furthermore, type of `quantized_float` is `double` but variable name includes `float`.
Calculation may not have problem but it seems inconsistent.
How about changing variable name or variable type? :)

Reply:
This code was copied from `QuantizeWithMinMaxPass`.

It'll be better to update the variable name in the separate PR.